### DISCUSSION TITLE
Localized accessibility & error descriptions

### DISCUSSION THREAD

**Discussion by mangerlahn** [KaTeX/KaTeX#3826] [Ideas]
Would you be open to setup the project to produce accessibility descriptions & error messages localized to the users operating system language, or is that something not really to be desired?

**Comment by edemaine** [comment]
This is an interesting idea and it would be good to get more input from users here (would you find this helpful?). Some initial thoughts:
* Maintenance might be difficult as we add/improve error messages (see e.g. #2133). That said, this evolution is probably fairly slow. But we'd need a maintainer for each language.
* We can't (usually) detect the operating system language, as KaTeX is usually run server-side. But it could be a configuration option.
* Adding more strings would increase standard bundle size (without having users make custom bundles). I'm not sure how many messages this is, though; needs measurement. But there are a lot of languages.
* Translating our documentation, maybe including a new page of what error messages mean (with translations), might be a better place to focus translation efforts? We don't yet have infrastructure in place for that either, but I imagine Docusaurus supports it; someone would need to take the first step.
